examples/plot_clip.py

```diff
@@ -0,0 +1,102 @@
+"""
+Clip Vector Data with EarthPy
+==================================================================
+
+Learn how to clip point, line, or polygon geometries with a
+polygon geometry using EarthPy.
+
+"""
+
+###############################################################################
+# Clip Vector Data in Python Using EarthPy
+# ---------------------------------------------
+#
```

Comment:
In the other vignettes, there is a note before the introductory paragraph (see the vignette on [NDVI](https://earthpy.readthedocs.io/en/latest/gallery_vignettes/plot_calculate_classify_ndvi.html#sphx-glr-gallery-vignettes-plot-calculate-classify-ndvi-py)). To modify this vignette to be more consistent with the others, I suggest: 
* add a new note before the existing paragraph lines 14-17 that says something like: "The example below will show you how to use the ```clip_shp()``` function to clip vector data such as points, lines and polygons to a vector boundary."
* add the sentence on lines 18-19 to the existing note about the multiple polygons, so that all the information regarding the extent is together.